### DISCUSSION TITLE
Custom Landscapes: Location values not set from landscape.ini

### DISCUSSION THREAD

**Discussion by briandoylegit** [Stellarium/stellarium#1207] [General]
Values input into the landscape.ini in the format of

[location]
latitude = +48d10'9.707"
longitude = +11d36'32.508"

are not displaying within stellarium

**Comment by gzotti** [comment]
Does it work with 2 decimals after the dot?
If not, please specify platform, or write exactly how you can discern that this fails.

**Comment by alex-w** [comment]
Please use dot as delimiter for fraction

**Comment by briandoylegit** [comment]
I don't seem to be able to upload a zipped horizon here.
I can email an example if required.
No reducing to 2 decimal places does not work.

Background:

Michael Kosowsky from www.heywhatsthat.com has just included export functionality from HWT to export Stellarium Polygonal Horizons.

Michael has tried using decimal Lat Lon to as location.

Stellarium guide suggests this format

latitude = +48d10'9.707"
longitude = +11d36'32.508"

If I substitute values as above into a landscape.ini location still does not update.

Andrew Smith's horizon program does output in the suggested format but also will not pull location values from the ini.

Example from horizon

[location]
planet = Earth
latitude = +53d45'21"
longitude = -8d16'1"
altitude = 120

**Comment by gzotti** [comment]
Great news about HWT's direct export, thanks to all involved. 

Are you sure you have activated "Use associated planet and position" in the landscape tab? I never had problems. Of course, you can also use decimal numbers.

**Comment by briandoylegit** [comment]
Hi Georg,

Many thanks for the mail.

I have played with this a bit and I can now see what is what is happening...

I import a new horizon. I can see the appropriate Lat Lon and Height values
in the description.
I select "Use associated planet and position"....

and there is no visual change on the interface. (but potentially the values
are being applied)

However if I then select a different horizon and then change back to the
original, the values are now applied (or present on the interface anyhow)

So clicking away and then clicking back fixes the issue.

Confusing but functional.



Re: HWT Stellarium Export

My only part played in this could be described as bashing a problem loudly
with a hammer until someone with the appropriate skillset and toolset shows
up.
In this instance that is.....

Michael Kosowsky
mk@heywhatsthat.com

Output from HWT can demonstrably be shown to be more accurate than Andrew
Smiths horizon program (for near horizon values) when compared to
photometric equivalents.

However output generated using 1 arc second srtm is limited to the USA
currently so there is a trade off there.



Thanks and regards

Brian

**Comment by gzotti** [comment]
You must have "Use associated planet and location" active before loading any landscape. 

If you are editing values in the landscape.ini and try to reload a landscape, it may be retrieved from cache. This may be confusing while developing. The intro of chapter 7 in the User Guide has explanations to switch off chaching, however recently we just had a report that this may not always work.

**Comment by briandoylegit** [comment]
Thanks Georg,

Apologies for messing up your bug log with my lack of knowledge.

While I have you online take a look at

https://sketchfab.com/3d-models/me026-033-henge-at-newgrange-baa82b53ec954876961347758305763d


This is a 3d model of the henge discovered at Newgrange 2 years ago.

I have a copy of this model and the creator Paul McNally is happy for it to
be used for research purposes where his name is retained as the creator. I
think this would make an excellent Stellarium model. Unfortunately I don't
have time at present to have a go at getting it to render in Stellarium.

Do you know anyone who might like to have a go at converting for use within
Stellarium ?

The model is in a standard format and is geocoded.

I think the only stipulation would be that the model be licensed
(appropriately) and made available on github (or equivalent) PRIOR to any
astronomic analysis if that is an intention.

Cheers

Brian

**Comment by gzotti** [comment]
Thanks for the model hint.

For Sketchup, most instructions for exporting are in the User Guide, or I could do the export myself. Georeferencing in Sketchup means the model origin has defined UTM coordinates (in inches...) and a north direction. It would of course be very valuable to add a topographic model of maybe 500m around the monument, and a surrounding panorama landscape (at least HWT or Horizon) seen from the centre of this circle, to get a feeling about the surrounding, else astronomical analysis is somewhat questionable.

We have not received a single model that we could host, so we don't have a repository yet. The author could provide a package on his own website, or allow us to host it on stellarium.org. For the latter case, I think (if @alex-w agrees) we could do that only under a CC license, this of course includes retaining his authorship, but we cannot made responsible for any inappropriate use.   

The model should come with sufficient description about culture, age, state of data collection, prospection/excavation/extrapolated assumptions etc. and further reading to make it an interesting contribution. Also, notes "what to see/demonstrate in Stellarium" are of course welcome.

**Comment by alex-w** [comment]
> For the latter case, I think (if @alex-w agrees) we could do that only under a CC license, this of course includes retaining his authorship, but we cannot made responsible for any inappropriate use.

Why CC only? Any free licenses...

**Comment by briandoylegit** [comment]
I am not a licensing expert but I think the MIT license may be appropriate.
But I am sure Paul will agree to any type you guys would suggest.

I will get Paul's agreement on a license.

I will get Paul's agreement you host.

I will generate at a minimum an HORIZON derived landscape using 1 arc
second SRTM

I will supply all detail re: description about culture, age, state of data
collection, prospection/excavation/extrapolated assumptions etc.

Re: The topographic element. A LIDAR set is available in the public
domain.  I myself have never attempted this type of model with LIDAR. I
have done it with single tile SRTM DEM. I suspect with LIDAR some form of
resolution reduction would be necessary or the file would be huge I think.
Let me see what I can do, or advise.

Thanks

Brian

**Comment by gzotti** [comment]
If you can provide source info (URL?) for the PD Lidar and something like a shapefile to find where the ring is, I could be tempted enough to do the terrain and assemble the models. Not tomorrow, but on a weekend in the near future. It depends on how accurate the landscape "has to be" to select SRTM1" or 1m or 5m Lidar. I create a triangulated mesh from that. The 3 of us could  then release it under a free license that you agree upon and host it at Stellarium.org for free use with Stellarium. 

The thing could need a texture for the landscape. Current aerial photo, or a prospection or excavation map, anything useful, whatever you have and can make available to the interested public.

**Comment by briandoylegit** [comment]
Hi Georg,

The LIDAR data is licensed for re-use under the Creative Commons
Attribution 4.0 International license.

Source URL's for the relevant LIDAR are available here.
https://dcenr.maps.arcgis.com/apps/webappviewer/index.html?id=b7c4b0e763964070ad69bf8c1572c9f5


The relevant block is the red coloured section on the east coast as
indicated in the attached image.
Click on the boxes to retrieve data url's.

WMS or REST available here if preferred.
https://data.gov.ie/dataset/open-topographic-lidar-data

Archaeological testing would suggest open pasture for any texture being
applied.

Shapefile to follow.

The henge is visible to Apple map users as their aerial imagery provider
captures the drought period when the henge was first observed by drone.
For non apple owners you can see the henge on aerial imagery here:
https://satellites.pro/Ireland_map#53.699045,-6.488136,15

What software are you using to convert LIDAR to 3d format ?

Cheers

Brian

**Comment by briandoylegit** [comment]
Actually this better represents where the henge actually is, that last url
is just a general view of Newgrange.

https://satellites.pro/Ireland_map#53.688206,-6.479176,18

Will the Lat-Lon in the string above suffice, or do you still need a
shapefile ?

B.

**Comment by gzotti** [comment]
Amazing area! I was not aware there are so many circle cropmarks in the fields around. 
I am using ArcGIS with my own TIN converter workflow. Seeing this and with the Lidar, I probably don't need a shapefile. If the Sketchup model is indeed georeferenced, it should then "just fit into place".

I cannot do it this or next week though. We should continue this in PM.

**Comment by briandoylegit** [comment]
https://www.archaeology.ie/sites/default/files/files/bru-na-boinne-interim-report.pdf


The site in question is referenced as site 5. in the document above.

You can send me PM details directly to brian.doyle.email@gmail.com

Yep, I don't have Arcgis only QGIS but theoretically it should be doable.

B.

**Comment by briandoylegit** [comment]
Hi Georg,

I asked Paul to slightly amend the model to

1. Remove the people

and

2. Create a second copy with no cross beams (as there is no archaeological
precedent for these)

I am ready to send them on, please advise on how to get these to you

Thanks

Brian

**Comment by gzotti** [comment]
(We continue in PM. I close this.)
